src/clusterfuzz/_internal/issue_management/google_issue_tracker/issue_tracker.py

```diff
@@ -48,6 +48,14 @@ class IssueAccessLevel(str, enum.Enum):
   LIMIT_VIEW_TRUSTED = 'LIMIT_VIEW_TRUSTED'
 
 
+def _get_access_limit_for_label(label: str):
+  """Returns IssueAccessLevel from string lable or None."""
+  try:
+    return IssueAccessLevel('LIMIT_' + label)
+  except:
+    return None
```

Comment:
should we log a warning/error in these cases?

Reply:
Done.